exercises/space-age/canonical-data.json

```
@@ -12,7 +12,16 @@
       "expected": 31.69
     },
     {
-      "description": "age on Mercury",
+      "description": "2 Earth years in Earth years",
+      "property": "age",
+      "input": {
+        "planet": "Earth",
+        "seconds": 63115200
+      },
+      "expected": 2
```

```suggestion
      "expected": 2.0
```

This is just something to make some test generators a bit more happy, the ones that discern between ints and float.